elastic/integrations
# [elasticsearch] Minor mappings updates

### Summary
Part of https://github.com/elastic/kibana/issues/172558

- add `cluster_settings.cluster.metadata.display_name` mapping
- remove `elasticsearch.index.created` mapping

### Testing
- set a cluster display_name
```
PUT /_cluster/settings
{
  "persistent": {
    "cluster.metadata.display_name": "custom cluster name"
  }
}
```
- verify field is correctly ingested and mapped in `cluster_stats` dataset
